app/services/auth_token_verifier.rb

```diff
@@ -3,9 +3,10 @@
 class AuthTokenVerifier
   attr_reader :errors
 
-  def initialize(token)
+  def initialize(token, member)
```

Comment:
Not clear why you're passing the member. This class no longer verifies the validity of the token.